client/bdf-font.cc

```diff
@@ -150,4 +150,16 @@ int DrawText(FlaschenTaschen *c, const Font &font,
     return x - start_x;
 }
 
+int VerticalDrawText(FlaschenTaschen *c, const Font &font,
+             int x, int y, const Color &color, const Color *background_color,
```

Comment:
The indentation looks off here; the parameters should align with the opening parenthesis above.
(sorry, I am a bit picky with code :) )